README.rst

```diff
@@ -3,26 +3,47 @@ MeerKAT Science Data Processor Telescope State
 
 This is a client package that allows connection to the Redis database that
 stores telescope state information for the Science Data Processor of the
-MeerKAT radio telescope.
+MeerKAT radio telescope. This database is colloquially known as *telstate*.
 
 The Redis database acts as a key-value store. Each key is either a *sensor* or
 an *attribute*. A sensor has multiple timestamped values organised into an
 ordered set. An attribute (or *immutable* key) has a single value without a
 timestamp that is not allowed to change.
 
-The keys are strings and values are arbitrary Python objects stored as pickles.
-Keys can be accessed using attribute syntax or dict syntax.
+The keys are strings and the values are Python objects serialised via
+MessagePack_. Older versions of the database stored the values as pickles, and
+the package warns the user if that's the case. Keys can be retrieved from the
+telstate object using attribute syntax or dict syntax.
+
+.. _MessagePack: http://www.msgpack.org/
```

Comment:
Maybe mention that it's been extended to support tuples, complex numbers and numpy arrays.

Reply:
Done.